tests/functions/test_origin_request.py

```diff
@@ -609,83 +649,46 @@ def test_origin_request_cookie_uri(
     ]
     assert request["headers"]["set-cookie"] == [
         {
-            "value": "CloudFront-Key-Pair-Id=K1MOU91G3N7WPY; Secure; "
-            "HttpOnly; SameSite=lax; Domain=ex.cloudfront.net; "
-            "Path=/content/; Max-Age=43200"
+            "value": "CloudFront-Key-Pair-Id=XXXXXXXXXXXXXX; Secure; "
+            "HttpOnly; SameSite=lax; Domain=http://localhost:8049; "
```

Comment:
The Domain changing to include the scheme might be a mistake.  That would probably be a bug in the exodus-gw change rather than here, but I've just noticed it here.

I don't actually know what the spec says, but e.g. at https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies the only example of Domain is `Domain=mozilla.org` - no scheme.

And the behavior we've implemented until now also doesn't include a scheme:

```
$ curl -Lv -b '' https://exodus-gw.dev.psi.redhat.com/rmcgover-pre/cdn/content/waf-test 2>&1 | grep -i Set-Cookie | tail -n1
< set-cookie: CloudFront-Signature=[...]; Secure; HttpOnly; SameSite=lax; Domain=dquugasaklmli.cloudfront.net; Path=/origin/; Max-Age=43200
```

Can you please check this?  We should probably not change this behavior if we don't have a specific reason to, e.g. if you found that a spec says we're doing it wrong.

Reply:
Ah, yeah, sounds good. I'll make that change in gw.